README.rst

```diff
@@ -256,25 +256,39 @@ Add content to repository ``foo``
 
 ``$ http POST ':24817'$REPO_HREF'versions/' add_content_units:="[\"$ROLE_HREF\"]"``
 
+.. code:: json
+
+    {
+        "task": "/pulp/api/v3/tasks/fd4cbecd-6c6a-4197-9cbe-4e45b0516309/"
+    }
+
+
+Create a Distribution for Repository 'foo'
+------------------------------------------
+
+This will distribute the 'latest' RepositoryVersion always.
 
-Create a Publication
--------------------------------------------------
+``$ http POST :24817/pulp/api/v3/distributions/ name='baz' base_path='dev' repository=$REPO_HREF``
 
-``$ http POST :24817/pulp/api/v3/publications/ansible/ansible/ repository=$REPO_HREF``
 
 .. code:: json
 
     {
-        "task": "/pulp/api/v3/tasks/fd4cbecd-6c6a-4197-9cbe-4e45b0516309/"
+        "_href": "/pulp/api/v3/tasks/2610a47e-4e88-4e8c-9d2e-c71734ae7b39/",
+       ...
     }
 
-``$ export PUBLICATION_HREF=$(http :24817/pulp/api/v3/publications/ansible/ansible/ | jq -r '.results[0] | ._href')``
 
 
-Create a Distribution for the Publication
------------------------------------------
+Create a Distribution for a RepositoryVersion
+---------------------------------------------
+
```

Comment:
Not sure how you prefer to do this, but in one of the PRs I suggest adding a link to 
https://github.com/pulp/pulpcore/pull/118/files#diff-9e0bbefde3f80fb5c48539015131cfefR38
https://github.com/pulp/pulpcore/pull/118/files#diff-9e0bbefde3f80fb5c48539015131cfefR64